README.md

```diff
@@ -53,20 +56,22 @@ git clone https://github.com/joshcanhelp/auth0-wp-test.git ./wp-content/themes/a
 # Optional, clones the testing theme to assist with development
 ```
 
```

Comment:
I feel this should be in the repo as an additional file, it seems strange for us to link to your own personal gist, appreciate there probably wasn't an accessible Auth0 one.

Reply:
I don't mind as long as the content is kept up to date.

Reply:
I totally agree. I was a little apprehensive to do it on my own one. It a lot of code to review to put out there, TBH. Would be nice as part of the plugin, though, as you could symlink from the `themes` folder (or copy) and use it for testing. I can add in another PR?